backend/src/services/secret/secret-fns.ts

```diff
@@ -309,7 +309,7 @@ export const interpolateSecrets = ({ projectId, secretEncKey, secretDAL, folderD
   };
 
   const expandSecrets = async (
-    secrets: Record<string, { value: string; comment?: string; skipMultilineEncoding?: boolean }>
+    secrets: Record<string, { value: string; comment?: string; skipMultilineEncoding?: boolean | null }>
```

Comment:
Why does our schema set skipMultilineEncoding to be nullable when the migration has it as a bool? @akhilmhdh

Reply:
Because it can be nullable. Default false doesn't mean it cannot be nullable.